src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java

```diff
@@ -31,21 +34,35 @@ public class InputOneStatementPerLine {
   ;; // violation 'Only one statement per line allowed.'
   static {
     new JCheckBox().addActionListener((final ActionEvent e) -> { good(); });
+    // violation above ''{' at column 64 should have line break after.'
     List<Integer> ints = new LinkedList<Integer>();
     ints.stream().map(t -> { return t * 2; }).filter(t -> { return false; });
+    // 2 violations above:
+    //  ''{' at column 28 should have line break after.'
+    //  ''{' at column 59 should have line break after.'
```

Comment:
If no related violation messages, please resolve them.

Reply:
reason behind violations: https://github.com/checkstyle/checkstyle/pull/15348#discussion_r1690157210

Reply:
ok, lets keep it.